google/ggrc-core
# GGRC-3942 Rename and add tip for default assessement type

# Issue description

1. Wrong naming  *Object under Assessment* in assessment template.
2. The  Default Assessment Type column doesn't have tip in csv template

# Steps to test the changes

1.Log into GGRC app.
2.Goto Export page.
3.Select the Object Type ( Assessment Templates) and export the csv file.
4.Open the excel and check for the *Default Assessment Type* column (The  column *Object under Assessment* is renamed )  , here you can see the tip message
"Allowed values are:
Access Groups
Account Balances
Data Assets
Facilities
Key Reports
Markets
Org Groups
Processes
Product Groups
Products
Systems
Technology Environments
Vendors
Contracts
Controls
Objectives
Policies
Regulations
Requirements
Risks
Standards
Threats
"

# Solution description

Below changes are done in src/ggrc/models/assessment_template.py
  1.Changed the  *template_object_type* attribute display_name to *Default Assessment Type*.
  2.Added the  description for *template_object_type* attribute.

# Sanity checklist




# PR Review checklist
